crates/xilem_core/src/view/rc.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2024 the Xilem Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#[macro_export]
+macro_rules! generate_rc_view {
+    ($($rc_ty:ident)::*, $viewtrait:ident, $viewmarker:ty, $cx:ty, $changeflags:ty; $($ss:tt)*) => {
+        impl<V> $viewmarker for $(::$rc_ty)*<V> {}
```

Comment:
Prefixing the paths with `::` in the output seems wrong, semantically. Instead, I would mirror the input expression exactly.

```suggestion
        impl<V> $viewmarker for $($rc_ty)::*<V> {}
```

Reply:
Well you're probably right, thanks